server/sandbox_run.go

```diff
@@ -80,7 +83,56 @@ func (s *Server) trustedSandbox(req *pb.RunPodSandboxRequest) bool {
 }
 
 func (s *Server) runContainer(container *oci.Container, cgroupParent string) error {
-	if err := s.Runtime().CreateContainer(container, cgroupParent); err != nil {
+	intermediateMountPoint := container.IntermediateMountPoint()
+
+	if intermediateMountPoint == "" {
```

Comment:
We could add some comments to explain why we are creating the intermediate mount point.

Reply:
yes, I will